docs/source/GetStartedGuide.rst

```diff
@@ -111,6 +111,22 @@ having the right compilers, header files and libraries available.  The
 more predictable and complete Avocado experience can be achieved with
 the official RPM packages.
 
+Build custom docker image
+--------------------------
+
+One of possible ways to develop avocado is to create docker image with your devel tree.
+This is good way to test your devel branch w/o breaking system instalation ::
+  # Fetch git source as usual
+  $ git clone github.com/avocado-framework/avocado.git avocado.git
+  $ cd avocado.git
+  # Make some changes
+  $ patch -p1 < MY_PATCH
+  # Finally build a docker image
+  $ docker build -t fedora-avocado-custom -f examples/install/Dockerfile.fedora .
+  # Run test inside custom docker image
+  $ avocado run --docker fedora-avocado-custom examples/tests/passtest.py
```

Comment:
Agree, one who want to use releases should use your official docker image. BTW: I've created automated image for debian users (avocado installed via python-pip ) https://hub.docker.com/r/dmonakhov/debian-avocado/  https://github.com/dmonakhov/debian-avocado . Feel free to include like, or simply copy debian's dockerfile 

But goal of this patch is to help developers to create docker images from scratch with modified avocado source. IMHO this greatly helps to devel new avocado features w/o breaking others.
For example: I want to implement some feature for avocado, but I already has avocado installed via 'make develop', so there is no way to play with devel feature w/o breaking existing instalation. So custom docker image is the best way to do so.
2) checkout avocado from git, apply my patches
3) build custom docker image
4) test my new feature (using unmodified host avocado instalation)
   $avocado run --docker my-test-image ./test-new-feature.py 
5) destroy custom image